crates/lib/src/compiler/isa/qubit.rs

```diff
@@ -170,7 +170,14 @@ impl FrbSim1q {
 const DEFAULT_DURATION_RX: f64 = 50.0;
 
 fn rx_gates(node_id: i64, frb_sim_1q: &FrbSim1q) -> Result<Vec<Operator>, Error> {
-    let fidelity = frb_sim_1q.fidelity_for_qubit(node_id)?;
+    let fidelity = match frb_sim_1q.fidelity_for_qubit(node_id) {
+        Ok(fidelity) => Ok(fidelity),
+        Err(Error::MissingBenchmarkForQubit(e)) => {
+            tracing::warn!(e);
```

Comment:
```suggestion
        Err(error @ Error::MissingBenchmarkForQubit(_)) => {
                    #[cfg(feature = "tracing")]
                    tracing::warn!(%error);
```

That resolves the "failed to resolve" check, and also makes the output from this:

```
WARN qcs::compiler::isa::qubit: e=15
```

to this:

```
WARN qcs::compiler::isa::qubit: error=Benchmark missing for qubit 15
```

Reply:
Great catch. This is the downside of using rust-analyzer with all features.

Fun little edge case - `error` is an unused binding if the `tracing` feature isn't enabled, which causes compilation errors. That is the first time I've had to legitimately use `#[allow(unused_variables)]`.